app/Talk.php

```diff
@@ -73,4 +73,18 @@ public function scopeActive($query)
     {
         return $query->where('is_archived', false);
     }
+
+    public function getSubmission(Conference $conference)
+    {
```

Comment:
I wonder if it would be cleaner/clearer to do:

```php
return Submission::where('conference_id', $conference->id)
    ->whereIn('talk_revision_id', $this->talkRevisions()->pluck('id'))
    ->first);
```

Not sure which is better tbh... my brain is a bit shot at the moment. I"ll just throw it out there and you can choose.

Same for getAcceptance below.

Reply:
I updated the function names to reflect that we're getting submissions for a specific user, which I don't believe your proposed query would do?

Reply:
👌